conn2res/connectivity.py

```diff
@@ -207,9 +250,12 @@ def get_nodes(self, node_set, nodes_from=None, nodes_without=None, filename=None
             # nodes we want to select from
             nodes_from = np.setdiff1d(nodes_from, nodes_without)
 
+            # use random number generator for reproducibility
+            rng = np.random.default_rng(seed=seed)
+
```

Comment:
I added seeds to all functions that use random numbers to enable reproducibility

Reply:
Amaziing! I should do the same for the other functions in the other modules. 
Thank you!